src/test/java/ca/bc/gov/ag/efax/mail/util/FaxUtilsTest.java

```diff
@@ -0,0 +1,15 @@
+package ca.bc.gov.ag.efax.mail.util;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class FaxUtilsTest {
+
+    @Test
+    public void getFaxDestinationTest() {
+       String faxDestination =  FaxUtils.getFaxDestination("IMCEAFAX", "testuser", "402505555555@gov.bc.ca");
+        Assertions.assertNotNull(faxDestination);
+        Assertions.assertEquals("402505555555@gov.bc.ca", faxDestination);
+    }
```

Comment:
This one, I might change the first parameter.  The actual fax format is something of the form:
IMCEAFAX-%RECIPIENT%+40%FAXNUMBER%@Victoria1.gov.bc.ca

This method is supposed to normalize the recipient (replace special characters, replace spaces with "+20", etc) and append the faxnumber to the recipient string.

So given:
format = "IMCEAFAX-%RECIPIENT%+40%FAXNUMBER%@Victoria1.gov.bc.ca"
recipent = "Someone at The Office"
faxNumber = "5551234567"

Result should be:
"IMCEAFAX-Someone+20at+20The+20Office+405551234567@Victoria1.gov.bc.ca"

You can expand this testing to confirm special characters are removed, or if the faxNumber has non-digits, etc.

It's actually debugging code that I found in there that if there is an @ symbol in the faxNumber, then don't do anything and assume the last param is an email address instead of a fax number.

Reply:
Thanks @KDNeufeld! this helps to understand better.